examples/qml_minimal/CMakeLists.txt

```diff
@@ -60,16 +60,21 @@ corrosion_set_env_vars(${CRATE}
     "QMAKE=${QMAKE}"
 )
 
+# Create an INTERFACE library target to link libraries to and add include paths.
+# Linking this to both the application and the tests avoids having to setup
+# the include paths and linked libraries for both of those.
+add_library(${CRATE}_interface INTERFACE)
```

Comment:
This actually is very similar to how Rust organizes executable building:
1. create a `lib` target that you link every dependency to (Qt, our Rust code, etc.)
2. Link that `lib` to both the executable, including the main file, as well as any tests.

So we should use that same analogy to explain what you're doing here, as that should be very easy to understand for any Rust developer.